cadc-sia/src/main/java/ca/nrc/cadc/sia2/impl/ServiceAvailability.java

```diff
@@ -86,29 +88,43 @@
  *
  * @author Sailor Zhang
  */
-public class ServiceAvailability implements WebService
+public class ServiceAvailability implements AvailabilityPlugin
 {
     private static final Logger log = Logger.getLogger(ServiceAvailability.class);
-    
+    private static final String DEFAULT_TAP_URI = "ivo://cadc.nrc.ca/tap";
```

Comment:
I don't think there should be the concept of a default URI in the code.

Reply:
Should it not be usable out of the box, though?  Should the default be put into the configuration, then?

Reply:
out sia service needs the config file like everyone else is OK

Reply:
Changed to check the class path first, then the external file system based on the `PropertiesReader` looks for external configuration.